src/python/grpcio_channelz/grpc_channelz/v1/_async.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""AsyncIO version of Channelz servicer."""
+
+from grpc.experimental import aio
+
+import grpc_channelz.v1.channelz_pb2 as _channelz_pb2
+import grpc_channelz.v1.channelz_pb2_grpc as _channelz_pb2_grpc
+from grpc_channelz.v1._servicer import ChannelzServicer as _SyncChannelzServicer
+
+
+class ChannelzServicer(_channelz_pb2_grpc.ChannelzServicer):
+    """AsyncIO servicer for handling RPCs for service statuses."""
+
+    @staticmethod
+    async def GetTopChannels(request: _channelz_pb2.GetTopChannelsRequest,
+                             context: aio.ServicerContext
+                            ) -> _channelz_pb2.GetTopChannelsResponse:
+        return _SyncChannelzServicer.GetTopChannels(request, context)
```

Comment:
That's not quite all it does. It gets the stats from core (which could itself constitute a blocking operation) and then [converts the JSON to a serialized protocol buffer message](https://github.com/grpc/grpc/blob/da428e90203d9e58f260833c6b725d3ba6463df4/src/python/grpcio_channelz/grpc_channelz/v1/_servicer.py#L31). For large json objects, this could take a while to complete, satisfying the (somewhat fuzzy) definition of "blocking operation." Again though, I think this is a problem that problem isn't high-value enough to invest the time to solve.

To clarify, the definition I'm using for "blocking" here is not "may not terminate for an arbitrarily long period of time". What I mean by "blocking" is "may take longer than some (unspecified) upper time bound." I would argue that this definition is more useful for us since `time.sleep(100)` doesn't meet the first definition of "blocking" but definitely ruins the responsiveness of your `asyncio` application.